website/content/docs/concepts/oidc-provider.mdx

```diff
@@ -0,0 +1,144 @@
+
+The next few sections of the document provide implementation details for each resource that permits Vault configuration as an OIDC identity provider.
+
+### Providers
+
+A Vault namespace may contain several provider resources. Each configured provider will publish the APIs listed within the OIDC flow. The APIs will be served via backend path-based routing on Vault's listen to address. The APIs will be served via the backend path-based routing on Vault's listen to the [address](/docs/configuration/listener/tcp#address).
+
+A provider must have the following configuration parameters:
+
+* **Issuer URL**: used in the `iss` claim of ID tokens
+* **Allowed client IDs**: limits which clients can access the provider
+* **Scopes supported**: limits what identity information is available as claims.
+
+The issuer URL parameter is necessary for the validation of ID tokens by clients. If an URL parameter is not provided explicitly, it will default to a URL with Vault's [api_addr](/docs/configuration#api_addr) as the `scheme://host:port` component and `/v1/:namespace/identity/oidc/provider/:name` as the path component. This means tokens issued by a provider in a specified Vault cluster must be validated within that same cluster. If the issuer URL is provided explicitly, it must point to a Vault instance that is network-reachable by clients for ID token validation.
+
+The allowed client IDs parameter utilizes the list of client IDs that have been generated by Vault as a part of client registration. By default, all clients will be *disallowed*. Providing an asterisk(*) as the parameter value will allow all clients to use the provider. The scopes parameter employs a list of references to named scope resources. The values provided are discoverable by the `scopes_supported` key in the OIDC discovery document of the provider. By default, a provider will have the `openid` scope available. See the scopes section below for more details on the `openid` scope.
+
+### Scopes
+
+Providers may reference scope resources via the `scopes_supported` parameter to make specific identity information available as claims.
+
+A scope will have the following configuration parameters:
+
+* **Description**: identity information captured by the scope
+* **Template**: maps individual claims to Vault identity information
+
+
+The template parameter takes advantage of the [JSON-based templating](/api-docs/secret/identity/tokens#template) used by identity tokens for claims mapping. This means the parameter will take a JSON string of arbitrary structure where the values may be replaced with specific identity information. Template parameters that are not present for a Vault identity are omitted from the resulting claims without an error.
+
+
+Example of a JSON template for a scope:
+
+```
+{
+  "groups": {{identity.entity.groups.names}},
+  "email": {{identity.entity.metadata.user_email}},
+  "auth0": {
+      "name": {{identity.entity.aliases.$MNT_ACC.metadata.user}}
```

Comment:
I am wondering if it makes sense to change `$MNT_ACC` to `$MOUNT_ACCESSOR` for more clarity?

Reply:
I missed this one initially. Yes, I think changing it to _ACCESSOR will provide that clarity. At first glance, I thought it might've been referring to "Account". I'll make that change.